lib/active_triples/rdf_source.rb

```diff
@@ -93,6 +93,8 @@ def initialize(*args, &block)
       resource_uri = args.shift unless args.first.is_a?(Hash)
       @rdf_subject = get_uri(resource_uri) if resource_uri
 
+      callback = args.first.delete(:callback) if args.first.is_a?(Hash)
```

Comment:
Can you add YARD documentation about this option. What sort of object is callback expected to be? Could it be renamed to `property_change_callback` or `on_property_change` to be more clear about it's purpose?